### PR TITLE
[docs] Document additional env variables used by `d8 mirror` family of commands

### PR DESCRIPTION
**Pull request by mvasl**
## Description

Added a list of env variables that `d8 mirror` understands as additional and rarely-used configuration options.

## Why do we need it, and what problem does it solve?

There was a request from internal teams for this information to be properly documented

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Add information about additional env variables used by `d8 mirror` command.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 1]
```diff
@@ -497,6 +497,14 @@ Deckhouse поддерживает работу только с Bearer token-с
...
 
+   Дополнительные параметры конфигурации для семейства команд `d8 mirror` доступны в виде переменных окружения:
+   - `HTTP_PROXY / HTTPS_PROXY` — URL-адреса прокси-сервера для запросов к HTTP(S) хостам, если они не указаны в `$NO_PROXY`;
+   - `NO_PROXY` — Список хостов, разделенных запятыми, которые следует исключить из проксирования. Каждое значение представлено в виде IP-адреса (1.2.3.4), CIDR (1.2.3.4/8), доменом или символом (*). IP-адреса и домены также могут включать номер порта (1.2.3.4:80). Доменное имя соответствует с и самому себе и всем поддоменам. Доменное имя начинающееся с "." соответствует только поддоменам. Например, «foo.com» соответствует «foo.com» и «bar.foo.com»; «.y.com» соответствует «x.y.com», но не соответствует «y.com». Символ `*` отключает проксирование;
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:502]:
```suggestion
   - `NO_PROXY` — Список хостов, разделенных запятыми, которые следует исключить из проксирования. Каждое значение представлено в виде IP-адреса (1.2.3.4), CIDR (1.2.3.4/8), доменом или символом (*). IP-адреса и домены также могут включать номер порта (1.2.3.4:80). Доменное имя соответствует как самому себе, так и всем поддоменам. Доменное имя, начинающееся с ".", соответствует только поддоменам. Например, «foo.com» соответствует «foo.com» и «bar.foo.com»; «.y.com» соответствует «x.y.com», но не соответствует «y.com». Символ `*` отключает проксирование;
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 2]
```diff
@@ -497,6 +497,14 @@ Deckhouse поддерживает работу только с Bearer token-с
...
+   - `SSL_CERT_FILE` — Указывает путь до сертификата SSL. Если переменная установлена, системные сертификаты не используются;
+   - `SSL_CERT_DIR` — Это список каталогов, разделенный двоеточиями, определяет в каких каталогах искать файлы сертификатов SSL. Если переменная установлена, системные сертификаты не используются. См. <https://www.openssl.org/docs/man1.0.2/man1/c_rehash.html>;
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:504]:
```suggestion
   - `SSL_CERT_DIR` — Список каталогов, разделенный двоеточиями; определяет, в каких каталогах искать файлы сертификатов SSL. Если переменная установлена, системные сертификаты не используются. См. <https://www.openssl.org/docs/man1.0.2/man1/c_rehash.html>;
```

**Review by kirkonru — APPROVED** [review]

**Review by Zhbert — APPROVED** [review]

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 3]
```diff
@@ -497,6 +497,14 @@ Deckhouse поддерживает работу только с Bearer token-с
...
 
+   Дополнительные параметры конфигурации для семейства команд `d8 mirror` доступны в виде переменных окружения:
+   - `HTTP_PROXY`/`HTTPS_PROXY` — URL-адреса прокси-сервера для запросов к HTTP(S) хостам, если они не указаны в `$NO_PROXY`;
+   - `NO_PROXY` — список хостов, разделенных запятыми, которые следует исключить из проксирования. Каждое значение представлено в виде IP-адреса (1.2.3.4), CIDR (1.2.3.4/8), доменом или символом (*). IP-адреса и домены также могут включать номер порта (1.2.3.4:80). Доменное имя соответствует как самому себе, так и всем поддоменам. Доменное имя, начинающееся с «.», соответствует только поддоменам. Например, «foo.com» соответствует «foo.com» и «bar.foo.com»; «.y.com» соответствует «x.y.com», но не соответствует «y.com». Символ `*` отключает проксирование;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:502]:
```suggestion
   - `NO_PROXY` — список хостов, разделенных запятыми, которые следует исключить из проксирования. Каждое значение может быть представлено в виде IP-адреса (`1.2.3.4`), CIDR (`1.2.3.4/8`), домена или символа (`*`). IP-адреса и домены также могут включать номер порта (`1.2.3.4:80`). Доменное имя соответствует как самому себе, так и всем поддоменам. Доменное имя начинающееся с `.`, соответствует только поддоменам. Например, `foo.com` соответствует `foo.com` и `bar.foo.com`; `.y.com` соответствует `x.y.com`, но не соответствует `y.com`. Символ `*` отключает проксирование;
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 4]
```diff
@@ -496,6 +496,14 @@ Check [releases.deckhouse.io](https://releases.deckhouse.io) for the current sta
...
 
+   Additional configuration options for the `d8 mirror` family of commands are available as environment variables:
+    - `HTTP_PROXY`/`HTTPS_PROXY` — proxy URLs for HTTP(S) requests unless overridden by `$NO_PROXY`;
+    - `NO_PROXY` — comma-separated list of hosts to exclude from proxying. Supported value formats include IP addresses (1.2.3.4), CIDR notations (1.2.3.4/8), domains, and the asterisk character (*).  The IP addresses and domain names can also include a literal port number (1.2.3.4:80). The domain name matches that name and all the subdomains. The domain name with a leading "." matches subdomains only. For example, "foo.com" matches "foo.com" and "bar.foo.com"; ".y.com" matches "x.y.com" but does not match "y.com". A single asterisk `*` indicates that no proxying should be done;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:501]:
```suggestion
    - `NO_PROXY` — comma-separated list of hosts to exclude from proxying. Supported value formats include IP addresses (`1.2.3.4`), CIDR notations (`1.2.3.4/8`), domains, and the asterisk character (`*`).  The IP addresses and domain names can also include a literal port number (`1.2.3.4:80`). The domain name matches that name and all the subdomains. The domain name with a leading `.` matches subdomains only. For example, `foo.com` matches `foo.com` and `bar.foo.com`; `.y.com` matches `x.y.com` but does not match `y.com`. A single asterisk `*` indicates that no proxying should be done;
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 5]
```diff
@@ -497,6 +497,14 @@ Deckhouse поддерживает работу только с Bearer token-с
...
+   - `SSL_CERT_FILE` — указывает путь до сертификата SSL. Если переменная установлена, системные сертификаты не используются;
+   - `SSL_CERT_DIR` — список каталогов, разделенный двоеточиями. Определяет, в каких каталогах искать файлы сертификатов SSL. Если переменная установлена, системные сертификаты не используются. [Подробнее...](https://www.openssl.org/docs/man1.0.2/man1/c_rehash.html);
+   - `TMPDIR (*nix)`/`TMP (Windows)` — путь к директории для временных файлов, который будет использоваться во время операций загрузки и выгрузки образов. Вся обработка выполняется в этом каталоге, он должен иметь достаточное количество свободного дискового пространства, чтобы вместить весь загружаемый пакет образов;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:505]:
```suggestion
   - `TMPDIR (*nix)`/`TMP (Windows)` — путь к директории для временных файлов, который будет использоваться во время операций загрузки и выгрузки образов. Вся обработка выполняется в этом каталоге. Он должен иметь достаточное количество свободного дискового пространства, чтобы вместить весь загружаемый пакет образов;
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 6]
```diff
@@ -497,6 +497,14 @@ Deckhouse поддерживает работу только с Bearer token-с
       - Для аутентификации в стороннем хранилище образов нужно использовать параметры `--source-login` и `--source-password`;
    - `--images-bundle-chunk-size=N` — для указания максимального размера файла (в ГБ), на которые нужно разбить архив образов. В результате работы вместо одного файла архива образов будет создан набор `.chunk`-файлов (например, `d8.tar.NNNN.chunk`). Чтобы загрузить образы из такого набора файлов, укажите в команде `d8 mirror push` имя файла без суффикса `.NNNN.chunk` (например, `d8.tar` для файлов `d8.tar.NNNN.chunk`).
 
+   Дополнительные параметры конфигурации для семейства команд `d8 mirror` доступны в виде переменных окружения:
+   - `HTTP_PROXY`/`HTTPS_PROXY` — URL-адреса прокси-сервера для запросов к HTTP(S) хостам, если они не указаны в `$NO_PROXY`;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:501]:
```suggestion
   - `HTTP_PROXY`/`HTTPS_PROXY` — URL-адрес прокси-сервера для запросов к HTTP(S) хостам, которые не указаны в списке хостов в переменной `$NO_PROXY`;
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 7]
```diff
@@ -496,6 +496,14 @@ Check [releases.deckhouse.io](https://releases.deckhouse.io) for the current sta
       - To authenticate in a third-party registry, you need to use the `--source-login` and `--source-password` parameters;
    - `--images-bundle-chunk-size=N` — to specify the maximum file size (in GB) to split the image archive into. As a result of the operation, instead of a single file archive, a set of `.chunk` files will be created (e.g., `d8.tar.NNNN.chunk`). To upload images from such a set of files, specify the file name without the `.NNNN.chunk` suffix in the `d8 mirror push` command (e.g., `d8.tar` for files like `d8.tar.NNNN.chunk`).
 
+   Additional configuration options for the `d8 mirror` family of commands are available as environment variables:
+    - `HTTP_PROXY`/`HTTPS_PROXY` — proxy URLs for HTTP(S) requests unless overridden by `$NO_PROXY`;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:500]: > proxy URLs
Is it a proxy URL or a list of proxy URLs?

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-FAQ.md:500]: @mvasl

Comment (mvasl) [reply docs/documentation/pages/DECKHOUSE-FAQ.md:500]: Single url

Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:500]:
```suggestion
    - `HTTP_PROXY`/`HTTPS_PROXY` — URL of the proxy server for HTTP(S) requests to hosts that are not listed in the variable `$NO_PROXY`;
```

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-FAQ.md:500]: Offered the suggestion.

**Review by z9r5 — APPROVED** [review]

**Pull request merged**